test/base/resources/eventing.go

```diff
@@ -0,0 +1,190 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resources
+
+// eventing contains functions that construct Eventing resources.
+
+import (
+	eventingv1alpha1 "github.com/knative/eventing/pkg/apis/eventing/v1alpha1"
+	pkgTest "github.com/knative/pkg/test"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// TriggerOption enables further configuration of a Trigger.
+type TriggerOption func(*eventingv1alpha1.Trigger)
+
+// SubscriptionOption enables further configuration of a Subscription.
+type SubscriptionOption func(*eventingv1alpha1.Subscription)
```

Comment:
I think Subscription and Channel should be in a messaging file. 
Although the apiGroup is still eventing, I think they are actually messaging. We haven't changed them as they will be deprecated. It's up to you, if you want you can leave them here. 
Broker and Trigger, yes, they are actually eventing.

Reply:
I would prefer to keep them here before the actual change.